packages/ui-popover/src/Popover/index.tsx

```diff
@@ -506,12 +452,14 @@ class Popover extends Component<PopoverProps> {
         // only set aria-expanded if popover can contain focus
         expanded = this.shown ? 'true' : 'false'
       } else {
-        expanded = null
+        expanded = undefined
       }
 
       trigger = safeCloneElement(trigger, {
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-        ref: (el) => (this._trigger = el),
+        // TODO: try to type `ref` better, LegacyRef<T> was not compatible
+        ref: (el: any) => {
+          this._trigger = el
+        },
```

Comment:
Typing the trigger was a nightmare, because it can be anything really. I also couldn't type it correctly, because it was always complaining about the LegacyRef not compatible with this callback. I' gladly accept any ideas how to do this better :)

Reply:
For me its accepting `ref: (el: React.ReactElement) => {`, isnt this OK?

Reply:
Yeah, I tried it before and was throwing errors, but works fine now, thanks :)